website/docs/basics/glossary.md

```diff
@@ -377,11 +377,14 @@ Cryptography is the practice and study of techniques for secure communication in
 ---
 
 ### Cycles
-The number of RISC-V computational cycles required by a script to execute. 
+The number of RISC-V computational cycles required by a script to execute. It's used to prevent malicious behaviors, such as a script with infinite loops, that's why it is called cycles. A script needs to pay attention not to consume too much cycle, otherwise the block will be rejected.
```

Comment:
The number of RISC-V computational steps required to execute script. It's a metric used to prevent malicious behavior such as infinite loops, and scripts must stay within cycle limits to avoid rejection. Similar to Ethereum's "Gas", we set cycles to ensures optimal performance and security.